content/docs/typechecking-with-proptypes.md

```diff
@@ -168,4 +170,23 @@ ReactDOM.render(
 );
 ```
 
+You can also put the defaultProps as static props inside React class.
```

Comment:
How do you feel about combining this paragraph and the one below it into something more like this:

> If you are using a Babel transform like [`transform-class-properties`](https://babeljs.io/docs/plugins/transform-class-properties/), you can also declare `defaultProps` as static property within a React component class. This syntax has not yet been finalized though and will require a compilation step to work within a browser. For more information, see the [class fields proposal](https://github.com/tc39/proposal-class-fields).

Reply:
Thank you for you help. It is a big challenge for a non-native English speaker to write in English.